mmv1/third_party/terraform/services/networkservices/resource_network_services_authz_extension_test.go

```diff
@@ -47,21 +47,21 @@ func TestAccNetworkServicesAuthzExtension_update(t *testing.T) {
 func testAccNetworkServicesAuthzExtension_start(context map[string]interface{}) string {
 	return acctest.Nprintf(`
 resource "google_compute_network" "default" {
-  name                    = "lb-network"
+  name                    = "lb-network%{random_suffix}"
```

Comment:
Should we also change these to `tf-test`...?

Reply:
good idea